### ISSUE TITLE
Click header containing tags does not toggle sorter

### ISSUE THREAD

**Issue by thaihacong** [primefaces/primeng#8597]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => Search github for a similar issue or PR before submitting
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/github-7e8q34

```
    <ng-template pTemplate="header" let-columns>
        <tr>
            <th *ngFor="let col of columns" [pSortableColumn]="col.field">
                <div class="custom-table-header">{{col.header}}</div>
                <p-sortIcon [field]="col.field" ariaLabel="Activate to sort" ariaLabelDesc="Activate to sort in descending order" ariaLabelAsc="Activate to sort in ascending order"></p-sortIcon>
            </th>
        </tr>
    </ng-template>
```

**Current behavior**
In my custom table, I have col.header wrapped inside a div to apply a custom style.
However, when I click on this div, column sorter is not toggled.
This worked in primeNG v8.1.1 but is broken in v9.0.0, after https://github.com/primefaces/primeng/commit/9abd2e4aebf3c9a8c557a18fa6cbf0463a97dfc1.

**Expected behavior**
Click on column header (title) should toggle sorter.

* **Angular version:** 9.0.4
* **PrimeNG version:** 9.0.0

**Comment by hugonoro** [comment]
I have the exact same issue after upgrading to version 9. After banging my head for a while on what was going on, I finally found this :). Curiously enough the Tree Table doesn't seem to be suffering from the same issue.

**Comment by cagataycivici** [comment]
Resolved for 9.0.1
